cmd/influxd/config.go

```diff
@@ -40,6 +40,10 @@ type Config struct {
 	Version           string `toml:"-"`
 	InfluxDBVersion   string `toml:"-"`
 
+	Authentication struct {
```

Comment:
Actually, it's not as bad as I first though, now that I look at it. Let me change it to the simpler form.